lib/lichess.py

```diff
@@ -279,44 +283,48 @@ def chat(self, game_id: str, room: str, text: str) -> JSON_REPLY_TYPE:
             logger.warning(f"This chat message is {len(text)} characters, which is longer "
                            f"than the maximum of {MAX_CHAT_MESSAGE_LEN}. It will not be sent.")
             logger.warning(f"Message: {text}")
-            return {}
+            return {"ok": False}
 
         payload = {"room": room, "text": text}
-        return self.api_post("chat", game_id, data=payload)
+        return cast(OKResponse, self.api_post("chat", game_id, data=payload))
 
-    def abort(self, game_id: str) -> JSON_REPLY_TYPE:
+    def abort(self, game_id: str) -> OKResponse:
         """Aborts a game."""
-        return self.api_post("abort", game_id)
+        return cast(OKResponse, self.api_post("abort", game_id))
 
+    # TODO: Align types with test_bot/lichess.py
```

Comment:
I'm pondering suggesting to make the `Lichess` class in `test_bot/lichess.py` subclass the one in `lib/lichess.py`, or they both subclass an abstract base class. But this type mismatch will be sorta a blocker to that idea.

Reply:
What benefit would there be to making subclasses? The two versions of the `get_event_stream()` method are only used here:
https://github.com/lichess-bot-devs/lichess-bot/blob/b3c9239f0a3bf40902c15cb3410279380c9d1d4c/lichess-bot.py#L96-L97
The type of `response` here is `EventStream | requests.models.Response` due to the two different classes returning different types for this method. However, since both return types have an `iter_lines()` method, the type checking passes. I don't see a need to align the return types of the `get_event_stream()` methods of `lib/Lichess` and `test_bot/Lichess`.

Reply:
OK sounds good, I will drop the TODO comment.

I have seen other mocking strategies in the past where they extended the actual implementation, but seems not necessary here. 👍